lib/instadeploy.py

```diff
@@ -169,3 +187,13 @@ def build():
         ))
 
     return response.json()
+
+
+def extract_mxbuild_response(mxbuild_response):
+    r = {}
+    if mxbuild_response and 'problems' in mxbuild_response:
+        r['buildstatus'] = json.dumps(mxbuild_response['problems'])
+        if 'message' in mxbuild_response:
```

Comment:
I also removed mxbuild_response None check as this is always going to be populated in the scope of this function